Challenges_week3/is_palindrome.rb

```diff
@@ -0,0 +1,14 @@
+# Write a method that takes a string and returns true if it is a
+# palindrome. A palindrome is a string that is the same whether written
+# backward or forward. Assume that there are no spaces; only lowercase
+# letters will be given.
+
+
+def is_palindrome(str)
+    if str == str.reverse
```

Comment:
It can be refactored with just single liner condition here. No need to explicitly return `true` and `false` inside `if - else` block